### DISCUSSION TITLE
Blockbench Plugin from the wiki website is broken

### DISCUSSION THREAD

**Discussion by tiffbu** [TartaricAcid/TouhouLittleMaid#528] [Default Model Feedback]
after clicking the "Download: Click Here" it says "This content is no longer available." on the page. i found a working link afterwords on this discussion, i just want to notify the one on the wiki is broken.

**Comment by Haobo-Ciao** [comment]
can please share the link? thank you

**Comment by tiffbu** [comment]
cdn.discordapp.com/attachments/760041309481336843/760044338239242240/tlm-utils.js
from  Blockbench Plugins introduction | Touhou Little Maid Wiki (cfpa.team)
<https://tlmwiki.cfpa.team/blockbench.html#what-is-blockbench>
[image: image.png]

**Reply by Haobo-Ciao** [reply]
Got it thank you!

**Comment by tiffbu** [comment]
i love the mod ^_^ 🫶🫶🫶

**Comment by TartaricAcid** [comment]
http://page.cfpa.team/TLM-Utils-Plugins/
